src/pages/components-gallery/index.js

```diff
@@ -1,12 +1,12 @@
 import { Container, SectionHeader } from '@/ui/containers';
-import { Gallery, Testimonials } from '@/ui/sliders';
+import { GallerySlider, Testimonials } from '@/ui/sliders';
 
-import { Form } from '@/components/ui/form';
+import { Form } from '@/ui/form';
 import { H4 } from '@/ui/typography';
 import Layout from '@/components/layout';
 import { useRouter } from 'next/router';
 
-const Components = () => {
+const ComponentsGallery = () => {
```

Comment:
This component uses a composition of components so it would be beneficial to write a docstring explaining what this ComponentGallery is used for.